### PR TITLE
New features

### PR DESCRIPTION
**Pull request by aseyfpour**

### CODE REVIEW

**Review by ebkhalilzadeh — COMMENTED** [review]
Do we have rate limit for getCharities API?

File: src/core/charityHelper.ts [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+        `Last balance is not up to date, new value is ${balance} SWASH`,
+      );
+    }
+
+    await storageHelper.saveProfile(profile);
+  }
+
+  async function startAutoPayment() {
+    paymentInterval && clearInterval(paymentInterval);
+    paymentInterval = setInterval(paymentToCharities, 5 * 60000);
```
Comment (ebkhalilzadeh) [inline src/core/charityHelper.ts:142]: It's better to keep the interval in the config file.

**Comment by aseyfpour** [comment]
> Do we have rate limit for getCharities API?

Yes we have

File: src/core/charityHelper.ts (hunk 1 above)
Comment (aseyfpour) [reply src/core/charityHelper.ts:142]: OK, I'll move them to a config file

**Review by ebkhalilzadeh — APPROVED** [review]

**Pull request merged**